### ISSUE TITLE
Add issue / PR to Things

### ISSUE THREAD

**Issue by bswinnerton** [bswinnerton/launchbar-github#92]
[Things](https://culturedcode.com/things/) has support for [URI schemes](https://support.culturedcode.com/customer/portal/articles/2803573). It'd be really neat if you could paste in a GitHub URL and have LaunchBar fetch the title via an API call (using the GraphQL API's top-level [`resource`](https://developer.github.com/v4/reference/interface/uniformresourcelocatable/) field), and then add it to Things.

In other words, when a user enters a URL like the following into LaunchBar:

`https://github.com/bswinnerton/launchbar-github/pull/91` 

...have it automatically convert it to something like: 

`Review bswinnerton/launchbar-github#91 "Use first class hide() function from LaunchBar"`

With the full link back to that Issue / PR.

**Comment by bswinnerton** [comment]
I wonder if this should be a different LaunchBar action 🤔 .

The commonality between the two is having an Action that is capable of hitting GitHub's GraphQL API.
